packages/opc-styles/functions/_custom-properties.scss

```diff
@@ -0,0 +1,109 @@
+/// Author: https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-sass/functions/_custom-properties.scss
+/// Returns the css variable with category type and fallback if provided
+///
+/// this function is called in the opc-color function.
```

Comment:
@gisumit check these inconsistencies in comments.

Reply:
removed the empty comment line.